modules/monitoring-setting-query-log-file-for-prometheus.adoc

```diff
@@ -3,14 +3,35 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="setting-query-log-file-for-prometheus_{context}"]
-= Enabling the query log file for Prometheus
 
-[role="_abstract"]
+// The ultimate solution DOES NOT NEED separate IDs and titles, it is just needed for now so that the tests will not break
```

Comment:
Preferably use persent tense, so could this be reworded as: "...needed for now so that test do not break"

Reply:
Because this is just for the reviewers, not user facing, I think I do not really need to deal with style guide here :)